File: src/main/java/MachTeacher/MachTeacher/dto/SessionDTO.java
package MachTeacher.MachTeacher.dto;

import java.time.LocalDate;
import MachTeacher.MachTeacher.model.Modality;
import MachTeacher.MachTeacher.model.SessionStatus;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SessionDTO {
    private Long id;
    private Long studentId;
    private Long mentorId;
    private Long subjectId;
    private Long packageTypeId;
    private LocalDate date;
    private String startTime;
    private int durationMinutes;
    private Modality modality;
    private SessionStatus status;
    private double priceUsd;
    private String notes;
    private String mentorName;
    private String subjectName;
}

File: MachTeacher/app/src/main/java/com/example/machteacher/model/SessionDto.kt
package com.example.machteacher.model

data class SessionDto(
    val id: Long,
    val studentId: Long,
    val mentorId: Long,
    val subjectId: Long,
    val packageTypeId: Long?,
    val date: String,
    val startTime: String,
    val durationMinutes: Int,
    val modality: String,
    val status: String,
    val priceUsd: Double,
    val notes: String?
)
